components/suggestion/command_provider/app_command_parser/app_command_parser.ts

```diff
@@ -839,7 +900,7 @@ export class AppCommandParser {
     // getChannel gets the channel in which the user is typing the command
     private getChannel = (): Channel | null => {
         const state = this.store.getState();
-        return getChannel(state, this.channelID);
+        return selectChannel(state, this.channelID);
```

Comment:
Definitely 0/5 on this, but I've been thinking of changing the actions to load objects from the server to be `fetchX` and leaving the selectors as `getX`. No actual plans to do that though

Reply:
I agree, `fetch` and `get` make the most sense to me